Applications/DataExplorer/DataView/GMSHPrefsDialog.cpp

```diff
@@ -147,16 +147,17 @@ void GMSHPrefsDialog::accept()
 
     if (this->radioAdaptive->isChecked())
     {
+        double const min_scaling_factor (1e-10);
         max_number_of_points_in_quadtree_leaf = BaseLib::str2number<unsigned> (
                 param1->text().toStdString().c_str());
         if (max_number_of_points_in_quadtree_leaf == 0)
             max_number_of_points_in_quadtree_leaf = 10;
         mesh_density_scaling_pnts = fabs (param2->text().toDouble());
-        if (mesh_density_scaling_pnts < sqrt(std::numeric_limits<double>::epsilon()))
-            mesh_density_scaling_pnts = 0.5;
+        if (mesh_density_scaling_pnts < min_scaling_factor)
+            mesh_density_scaling_pnts = min_scaling_factor;
```

Comment:
In the previous version the value was 10 x larger than the value for `mesh_density_scaling_stations`. Now, `mesh_density_scaling_pnts` equals `mesh_density_scaling_stations`. Is this change intended?

Reply:
Yes, it is intended. The smallest possible number allowed by the DoubleValidator is "0", but the algorithm requires a small positive value. So instead of setting it to the (relatively large) default value, it is now set to the previously allowed minimum value of 1^{-10}.